robosherlock/config/config_kinect_robot_tracking.ini

```diff
@@ -0,0 +1,20 @@
+[camera]
```

Comment:
delete this file. There are way too many config files already in the core; I guess it is only the filterBlurred images that was changed; we can put that in the documentation